app/src/main/res/layout/topic_card.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
```

Comment:
Hadn't noticed that it was actually using a CardView. Is this necessary? It doesn't look like a Card and all the card elements are hidden by using elevation = 0 and a transparent background.

Change the CardView to a LinearLayout and it will look the same.